muikku/src/main/webapp/resources/theme-muikku/sass/flex/workspace-materials.scss

```diff
@@ -73,23 +73,51 @@ body {
   }
   
   // Alternative Section's titles
-  & > h3, > h4, > h5, > h6 {
-    position: relative;
-    z-index: 13;
-    width: auto;
+  & > h3, > h4, > h5, > h6 {  
```

Comment:
Cannot test these, because can't create new material in dev and that property limit is just stupid, but will google it and get back to that.